changelogs/fragments/3216-nmcli-idempotency-fixes.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - fixed idempotency issue when modifying bridge connections - module forces default value of bridge.priority to nmcli; when bridge.stp is disabled nmcli ignores is and keep default (https://github.com/ansible-collections/community.general/issues/3216, https://github.com/ansible-collections/community.general/issues/4683)
```

Please note that fragments should look like this:
```suggestion
  - nmcli - fixed idempotency issue when modifying bridge connections. Module forces default value of bridge.priority to nmcli; when bridge.stp is disabled nmcli ignores is and keep default (https://github.com/ansible-collections/community.general/issues/3216, https://github.com/ansible-collections/community.general/issues/4683).
```

---

Ok, I will correct all fragments

---

I corrected all fragments in new PRs